src/main/twirl/views/manage/previewOppSection.scala.html

```diff
@@ -0,0 +1,36 @@
+@(
+opp: Opportunity,
+sectionNumber: Int
+)
+
+@formatId(id: Long) = @{
+f"RIFS$id%05d"
```

Comment:
This format has changed.  It's also been moved to `twirl/views/partials/formatId.scala.html`